packages/react-next/src/components/ContextualMenu/examples/ContextualMenu.Checkmarks.Example.tsx

```diff
@@ -167,7 +167,7 @@ export const ContextualMenuCheckmarksExample: React.FunctionComponent = () => {
           iconProps: {
             iconName: selection[keys[13]] ? 'SingleBookmarkSolid' : 'SingleBookmark',
           },
-          name: selection[keys[13]] ? 'Toogled command no checkmark' : 'Toogle command no checkmark',
+          name: selection[keys[13]] ? 'Toggled command no checkmark' : 'Toggle command no checkmark',
```

Comment:
Can you revert this one to avoid a conflict with the example move PR? (I'll fix it there)